projects/saturn/application.yaml

```diff
@@ -59,3 +59,11 @@ viewDatabase:
 search:
   pageRequestTimeout: 10000
   countRequestTimeout: 60000
+caches:
```

Comment:
I wonder if this should be a feature toggle, it is extra code and configuration while I assume it will always be activated.

What do you think @ewelinagr ?

Reply:
Well, to be honest I also think that it will be enabled permanently as we definitely won't have memory problems for this amount of cached data. And yes, we have to add a bunch of if's in the code to support the toggling functionality. Personally, I would get rid of toggle but left configuration as it might be extended one day in the future (I don't believe it, but this is still a good practice :) )

@ewelinagr could you please share your vision?

Reply:
To me it is also fine to keep the caching permanently enabled, the only comment I had around config for it was to make the time of automatic refresh of caches configurable (in the first draft it was hardcoded and set to daily refresh) and I still think this auto-refresh setup part should not be fixed

Reply:
I agree. Also, I don't think it has to be done per cache type (separate per facets, separate per views), can be single config for all the caches

Reply:
since now caches are enabled permanently with no toggle